src/patient-registration/patient-registration.component.tsx

```diff
@@ -529,7 +529,7 @@ export const PatientRegistration: React.FC = () => {
                       </div>
                     ))}
                     <Button style={{ marginBottom: '1rem', width: '11.688rem', display: 'block' }} type="submit">
-                      {existingPatient ? t('updatePatient') : t('registerPatient')}
+                      {existingPatient ? t('updatePatient') : t('Create Patient')} {/* createPatient */}
```

Comment:
The translation key was changed but not the actual translation. Could you change this instead and revert all the key changes? Thanks.

Reply:
Where are the translation keys stored? I tried searching for them, but couldn't find it.
No problem, I'll revert all the changes then. Register patient button was to be changed to Create Patient, that's why I made the changes. I'll make the changes directly to the key value then. 

Thanks.

Reply:
https://github.com/openmrs/openmrs-esm-patient-registration/blob/master/translations/en.json

Reply:
Thank You!